src/data/dialogmote/dialogmoteTypes.ts

```diff
@@ -1,3 +1,42 @@
+interface DialogmotedeltakerArbeidstakerVarselDTO {
```

Comment:
Usikker på om frontend-DTOene trenger å ha definert felter den ikke bruker (som f.eks. pdf). Nå vet vi kanskje ikke helt hva som skal brukes enda, men tror jeg foretrekker at typene i frontend bare inneholder det som er relevant for frontend... men dette kan sikkert diskuteres 😉

Reply:
Sant nok, nå tok jeg bare en kopi (mer eller mindre) av backend sin DTO. Kan fjerne litt av det